tests/integration/test_database_iceberg/test.py

```diff
@@ -609,3 +611,86 @@ def test_table_with_slash(started_cluster):
     create_clickhouse_iceberg_database(started_cluster, node, CATALOG_NAME)
     node.query(f"INSERT INTO {CATALOG_NAME}.`{root_namespace}.{table_encoded_name}` VALUES (NULL, 'AAPL', 193.24, 193.31, tuple('bot'));", settings={"allow_experimental_insert_into_iceberg": 1, 'write_full_path_in_iceberg_metadata': 1})
     assert node.query(f"SELECT * FROM {CATALOG_NAME}.`{root_namespace}.{table_encoded_name}`") == "\\N\tAAPL\t193.24\t193.31\t('bot')\n"
+
+
+def test_cluster_joins(started_cluster):
```

Comment:
This test seems to be passing on the current Antalya branch:

```
test_database_iceberg/test.py::test_cluster_joins PASSED  
```

Is it expected?

Reply:
Done.